packages/driver/src/cypress.js

```diff
@@ -152,6 +153,8 @@ class $Cypress {
 
     this.Cookies = $Cookies.create(config.namespace, d)
 
+    this.ProxyLogging = new ProxyLogging(this)
```

Comment:
What's the reason for exposing this on Cypress? Will the methods on `Cypress.ProxyLogging` being publicly documented?

Reply:
No, but it is accessed from `cy.route/cy.intercept`. What would be a better place?

Reply:
We could just not expose it for the time-being (make it a `const` instead of attaching it to `Cypress`), as long as users don't have need for accessing it.

Reply:
But yeah, we'd have to come up with a different way to pass them to `cy.route/intercept`...

Reply:
I'm ok with keeping it like this if you are, I kinda thought part of the reason `Cypress` is passed around when commands are registered *is* to let commands access utilities and global state like this.

Reply:
Yeah, I don't love how that makes it publicly-accessible for something we're not going to document, but it's consistent with what we've done up until now, so I'm okay with it.